src/components/ProductDetails/ProductDetails.tsx

```diff
@@ -135,6 +135,7 @@ export default function ProductDetails() {
             }}
           >
             <AddToCartSvg />
+            {quantity > 0 ? <span>{quantity}</span> : null}
```

Comment:
I mean in the aria-label

Reply:
Oh, I understand now. Sorry for that. Removed the span and added dynamic quantity to aria-label.